Failed tests (10971): TestStopperMultipleStopees

The following test appears to have failed:

[#10971](https://circleci.com/gh/cockroachdb/cockroach/10971):

```
=== RUN   TestStopper
--- PASS: TestStopper (0.01s)
=== RUN   TestStopperIsStopped
--- PASS: TestStopperIsStopped (0.01s)
=== RUN   TestStopperMultipleStopees
--- FAIL: TestStopperMultipleStopees (0.01s)
	stopper_test.go:126: timed out waiting for stop
=== RUN   TestStopperStartFinishTasks
I0111 11:53:32.657200 1251 stopper.go:264  draining; tasks left:
1      util/stop/stopper_test.go:143
--- PASS: TestStopperStartFinishTasks (0.00s)
	leaktest.go:110: prior leak detected, leaktest disabled
=== RUN   TestStopperRunWorker
--- PASS: TestStopperRunWorker (0.00s)
	leaktest.go:110: prior leak detected, leaktest disabled
=== RUN   TestStopperQuiesce
--
=== RUN   TestStopperShouldDrain
I0111 11:53:32.684238 1251 stopper.go:264  draining; tasks left:
1      util/stop/stopper_test.go:338
--- PASS: TestStopperShouldDrain (0.00s)
	leaktest.go:110: prior leak detected, leaktest disabled
FAIL
FAIL	github.com/cockroachdb/cockroach/util/stop	0.094s
=== RUN   TestContext
--- PASS: TestContext (0.00s)
=== RUN   TestTracer
--- PASS: TestTracer (0.00s)
=== RUN   TestNilTracer
--- PASS: TestNilTracer (0.00s)
=== RUN   TestEpochBalance
--- PASS: TestEpochBalance (0.00s)
PASS
ok  	github.com/cockroachdb/cockroach/util/tracer	0.024s

```
Please assign, take a look and update the issue accordingly.

--- a/util/stop/stopper_test.go
+++ b/util/stop/stopper_test.go
@@ -120,11 +120,7 @@ func TestStopperMultipleStopees(t *testing.T) {
 		close(done)
 	}()
 
-	select {
-	case <-done:
-	case <-time.After(10 * time.Millisecond):
-		t.Errorf("timed out waiting for stop")
-	}
+	<-done
 }
 
 func TestStopperStartFinishTasks(t *testing.T) {

